sdk/attestation/azure-security-attestation/azure/security/attestation/aio/_client_async.py

```diff
@@ -357,21 +357,22 @@ async def attest_open_enclave(
         )
 
     @distributed_trace_async
-    async def attest_tpm(self, content: str, **kwargs: Any) -> str:
+    async def attest_tpm(self, content: bytes, **kwargs: Any) -> TpmAttestationResult:
```

Comment:
If support for bytes is important here, we could do something like:
```python
ContentDataType = TypeVar("ContentDataType", bound=str|bytes)

async def attest_tpm(self, content: ContentDataType, **kwargs: Any) -> ContentDataType:
    try:
        # Input is str, so return str
        response = await self._client.attestation.attest_tpm(content.encode("ascii"), **kwargs)
        return response.data.decode("ascii")
    except AttributeError:
        # Input is bytes, so return bytes
        response = await self._client.attestation.attest_tpm(content, **kwargs)
        return response.data
```
(I haven't tested this - might need some refinement)